$NetBSD$

Add NetBSD, not tested at all, you need chunk for NetBSD in the bottom of this file.

--- src/VBox/HostDrivers/VBoxNetFlt/Makefile.kmk.orig	2015-07-10 23:31:29.000000000 +0900
+++ src/VBox/HostDrivers/VBoxNetFlt/Makefile.kmk	2015-09-23 21:23:48.000000000 +0900
@@ -313,7 +313,7 @@ $(PATH_TARGET)/VBoxNetAdpCat.dir/VBoxNet
 
 endif #ifeq ($(KBUILD_TARGET), win)
 
-else if1of ($(KBUILD_TARGET), linux solaris freebsd) ## @todo merge this with the mixed case stuff.
+else if1of ($(KBUILD_TARGET), linux solaris netbsd freebsd) ## @todo merge this with the mixed case stuff.
 #
 # vboxnetflt(.ko/.o/) - The lower case driver.
 # Note! On Solaris the name has to be <= 8 chars long.
@@ -342,6 +342,7 @@ else if1of ($(KBUILD_TARGET), linux sola
   vboxnetflt_SOURCES.linux = linux/VBoxNetFlt-linux.c
   vboxnetflt_SOURCES.solaris = solaris/VBoxNetFlt-solaris.c
   vboxnetflt_SOURCES.freebsd = freebsd/VBoxNetFlt-freebsd.c
+  vboxnetflt_SOURCES.netbsd = freebsd/VBoxNetFlt-freebsd.c
   vboxnetflt_SOURCES = VBoxNetFlt.c
   solaris/VBoxNetFlt-solaris.c_DEFS = VBOX_SVN_REV=$(VBOX_SVN_REV)
 
@@ -399,8 +400,9 @@ endif # linux
 
 # Gross hack for FreeBSD 7, should figure this out properly
 ## @todo Properly generate opt_netgraph.h
-ifeq ($(KBUILD_TARGET),freebsd)
+if1of ($(KBUILD_TARGET), freebsd netbsd) # this ifeq must go, see @todo around elif
  vboxnetflt_DEPS.freebsd += $(PATH_OUT)/opt_netgraph.h
+ vboxnetflt_DEPS.netbsd += $(PATH_OUT)/opt_netgraph.h
 $(PATH_OUT)/opt_netgraph.h:
 	echo > $(PATH_OUT)/opt_netgraph.h
 
